component/pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright (c)  2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~ Copyright (c)  2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We shouldn't change the licence year

Reply:
Reverted the change in [#e2a578ad1cf7ad69afdc70c95c019f2b5977913f](https://github.com/wso2-extensions/siddhi-execution-streamingml/pull/48/commits/e2a578ad1cf7ad69afdc70c95c019f2b5977913f)